pkgs/top-level/all-packages.nix

```diff
@@ -117,6 +117,8 @@ with pkgs;
 
   cmark = callPackage ../development/libraries/cmark { };
 
+  certmgr = callPackage ../tools/security/certmgr { };
```

Comment:
Can you move this to a more relevant section into `all-packages.nix`? Maybe line 1700. (Not blaming you though, that file is a mess..)

Reply:
Thanks. Done now. Yeah, would have been slightly better if it was just a single big sorted list.